#Expyriment 0.9.0 (Python 3.6.5), .xpe-file, coding: cp1252
#date: Wed Mar 27 2019 11:50:54
#sha1: None
#modules: 
#display: size=(1920, 1080), window_mode=False, open_gl=2
#os: uname_result(system='Windows', node='DESKTOP-OVOASGR', release='10', version='10.0.17134', machine='AMD64', processor='Intel64 Family 6 Model 94 Stepping 3, GenuineIntel')
Time,Type,Event,Value,Detail,Detail2
3896,stderr,received,'Traceback (most recent call last):'
3897,stderr,received,'  File "scripts\\MTTexp.py", line 393, in <module>'
3898,stderr,received,'    expyriment.design.Block.shuffle_trials(self = block)'
3900,stderr,received,"NameError: name 'block' is not defined"
